src/test/TestUtils/DotNetCli.cs

```diff
@@ -53,6 +53,19 @@ public DotNetCli(string binPath)
                 .First();
         }
 
+        public Command Run(params string[] args)
```

Comment:
Can we just change Exec to take an array only?
It doesn't really do anything different. 
Having both `Run` and `Exec` makes it confusing which one to use where (in reality it doesn't matter).